WalletWasabi.Fluent/Controls/PrivacyContentControl.axaml

```diff
@@ -8,24 +8,25 @@
   <Style Selector="controls|PrivacyContentControl">
     <Setter Property="Template">
       <ControlTemplate TargetType="controls:PrivacyContentControl">
-        <Panel Background="Transparent">
+        <Panel>
```

Comment:
I am wondering if couldn't we set the `MinHeight` here depending on the `ContentPresenter`'s height? Instead of setting them one by one everywhere.

Reply:
@soosr that means we should set the `ContentPresenter` to `Hidden` instead of `Collapsed` (sorry for the WPF terminology here). Not sure that's a good idea though

Reply:
Setting min values kind of bothers me... the content can change anytime and I am sure the MinHeights won't be adjusted.